pkg/csi/cinder/driver.go

```diff
@@ -70,12 +70,14 @@ func NewDriver(nodeID, endpoint, cluster string) *CinderDriver {
 			csi.ControllerServiceCapability_RPC_PUBLISH_UNPUBLISH_VOLUME,
 			csi.ControllerServiceCapability_RPC_CREATE_DELETE_SNAPSHOT,
 			csi.ControllerServiceCapability_RPC_LIST_SNAPSHOTS,
+			csi.ControllerServiceCapability_RPC_EXPAND_VOLUME,
 		})
 	d.AddVolumeCapabilityAccessModes([]csi.VolumeCapability_AccessMode_Mode{csi.VolumeCapability_AccessMode_SINGLE_NODE_WRITER})
 
 	d.AddNodeServiceCapabilities(
 		[]csi.NodeServiceCapability_RPC_Type{
```

Comment:
I think we also need to advertise `VolumeExpansion ONLINE`, in `GetPluginCapabilites`
https://github.com/container-storage-interface/spec/blob/master/spec.md#getplugincapabilities
https://github.com/kubernetes/cloud-provider-openstack/blob/master/pkg/csi/cinder/identityserver.go#L58

Reply:
Thanks. will update